mhkit/wave/io/hindcast/hindcast.py

```diff
@@ -170,6 +171,7 @@ def request_wpto_point_data(
     assert isinstance(str_decode, bool), 'str_decode must be bool type'
     assert isinstance(hsds, bool), 'hsds must be bool type'
     assert isinstance(path, (str, type(None))), 'path must be a string'
+assert is instance(as_xarray, bool), 'as_xarray must be bool type'
```

Comment:
Looks like an indention error is causing the tests to fail.

Reply:
@ssolson My bad, my GitHub suggestion wasn't robust. I fix this and double checked the others. Should work now